velox/common/encode/Base64.cpp

```diff
@@ -194,8 +225,6 @@ template <class T>
   auto wp = out;
   auto it = data.begin();
 
-  // For each group of 3 bytes (24 bits) in the input, split that into
```

Comment:
Why remove these comments. Below as well.

Reply:
revereted the changes